src/site/content/en/blog/one-line-layouts/index.md

```diff
@@ -0,0 +1,316 @@
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/1linelayouts?path=README.md&previewSize=100"
+    alt="1 Line Layouts"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+To follow along or play with these demos on your own, check out the Glitch embed above, or visit [1linelayouts.glitch.me](1linelayouts.glitch.me).
+
+## 01. Super Centered: `place-items: center`
+
+<figure class='w-figure'>
+<video controls autoplay loop muted class='w-screenshot'>
+    <source src='https://storage.googleapis.com/web-dev-assets/one-line-layouts/01-place-items-center.mp4'>
+  </video>
+</figure>
+
+For the  first 'single-line' layout, let's solve the biggest mystery in all of the CSS land: centering things. I want you to know that it's easier than you think with [`place-items: center`](https://developer.mozilla.org/en-US/docs/Web/CSS/place-items).
+
+First specify `grid` as the `display` method, and then write `place-items: center` on the same element. `place-items` is a shorthand to set both `align-items` and `justify-items` at once. By setting it to `center`, both `align-items` and `justify-items` are set to  `center`.
+
+```css/2
+.parent {
+  display: grid;
+  place-items: center;
+}
+```
+
+This enables the content to be perfectly centered within the parent, regardless of intrinsic size.
+
+## 02. The Deconstructed Pancake: `flex: <grow> <shrink> <baseWidth>`
+
+<figure class='w-figure'>
+<video controls autoplay loop muted class='w-screenshot'>
+    <source src='https://storage.googleapis.com/web-dev-assets/one-line-layouts/02-deconstructed-pancake-1.mp4'>
+  </video>
+</figure>
+
+Next we have the deconstructed pancake! This is a common layout for marketing sites, for example, which may have a row of 3 items, usually with an image, title, and then some text, describing some features of a product. On mobile, we'll want those to stack nicely, and expand as we increase the screen size. 
+
+By using Flexbox for this effect, you won't need media queries to adjust the placement of these elements when the screen resizes.
+
+The [`flex`](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) shorthand stands for: `flex: <flex-grow> <flex-shrink> <flex-basis>`.
+
+Because of this, if you want your boxes to fill out to their `<flex-basis>` size, shrink on smaller sizes, but not *stretch* to fill any additional space, write: `flex: 0 1 <flex-basis>`, setting the `<flex-grow>` to 0. In this case, your `<flex-basis`> is `'150px'` so it looks like:
+
+```css/5
+.parent {
+  display: flex;
+}
+
+.child {
+  flex: 0 1 150px;
+}
+```
+
+If you *do* want the boxes to stretch and fill the space as they wrap to the next line, set the `'flex-grow'` to `'1'`, so it would look like:
```

Comment:
```suggestion
If you *do* want the boxes to stretch and fill the space as they wrap to the next line, set the `<flex-grow>` to `1`, so it would look like:
```